blockchain/stakeversion.go

```diff
@@ -70,55 +70,6 @@ func (b *BlockChain) findStakeVersionPriorNode(prevNode *blockNode) *blockNode {
 	return prevNode.Ancestor(wantHeight)
 }
 
-// isVoterMajorityVersion determines if minVer requirement is met based on
-// prevNode.  The function always uses the voter versions of the prior window.
-// For example, if StakeVersionInterval = 11 and StakeValidationHeight = 13 the
-// windows start at 13 + 11 -1 = 24 and are as follows: 24-34, 35-45, 46-56 ...
-// If height comes in at 35 we use the 24-34 window, up to height 45.
-// If height comes in at 46 we use the 35-45 window, up to height 56 etc.
-//
-// This function MUST be called with the chain state lock held (for writes).
-func (b *BlockChain) isVoterMajorityVersion(minVer uint32, prevNode *blockNode) bool {
```

Comment:
This shoudn't be removed since it's used by the tests to make sure things are working properly.  It can, however, be moved to `stakeversion_test.go` so it is only compiled when running the tests.